contracts/scoring/dsf.sol

```diff
@@ -0,0 +1,35 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+import { IScoringFunction } from "../interface/ScoringFunction.sol";
+import { HashingHub } from "../HashingHub.sol";
+import { Hub } from "../Hub.sol";
+
+contract DSF is IScoringFunction {
+    Hub public hub;
+
+    constructor(address hubAddress) {
+        require(hubAddress != address(0));
+        hub = Hub(hubAddress);
+    }
+
+    function calculateScore(uint256 distance, uint96 stake, uint32 a, uint32 b)
```

Comment:
in this case "a" and "b" should be variables in the contract, which should also have their respective setters (protected by onlyOwner)

Reply:
Done